plugins/notifications-backend/src/service/router.ts

```diff
@@ -23,7 +25,14 @@ import { RouterOptions } from './types';
 export async function createRouter(
   options: RouterOptions,
 ): Promise<express.Router> {
-  const { logger, dbConfig, catalogClient } = options;
+  const { logger, database, discovery, config } = options;
+
+  // workaround for creating the database when client is not sqlite
```

Comment:
Is this a normal thing?  Is there an issue for this somewhere?

Reply:
Nope.  I fixed that in the upstream PR:
https://github.com/backstage/backstage/pull/22099/files#diff-72c8723c8f4d71d407e124d229ed18258167dc4547c9babc7ac3faac3226b413R36

and plan to cherry-pick the change to this PR as well.
But in an independent PR.